test/monitoring/aws-dynamo/DynamoTableGlobalSecondaryIndexMonitoring.test.ts

```diff
@@ -22,6 +22,16 @@ test("snapshot test: no alarms", () => {
   const monitoring = new DynamoTableGlobalSecondaryIndexMonitoring(scope, {
     table,
     globalSecondaryIndexName: "non-existing-index",
+    addReadThrottledEventsCountAlarm: {
+      Warning: {
+        maxThrottledEventsThreshold: 5,
+      },
+    },
+    addWriteThrottledEventsCountAlarm: {
+      Warning: {
+        maxThrottledEventsThreshold: 5,
+      },
+    },
   });
```

Comment:
You'll be able to correctly assert that 2 alarms were created here too ([example](https://github.com/cdklabs/cdk-monitoring-constructs/blob/8999ae6cedc3648a82e9b31c0e78e342dbc2bdd7/test/monitoring/aws-dynamo/DynamoTableMonitoring.test.ts#L128)).

Reply:
Added another test case and the assertion.